### DISCUSSION TITLE
nr ITS in Polyploid Lineages

### DISCUSSION THREAD

**Discussion by DKoenemann** [Kinggerm/GetOrganelle#304] [Q&A]
Does the workflow that assembles the nr ITS (embplant_nr) have a mechanism for sorting multiple copies in polyploid lineages? For example, in a tetraploid, under the best case data scenario, and with four copies of ITS with different sequences, would GetOrganelle be able to retrieve the four different sequences? Or will it attempt to report a single consensus sequence? Thank you.

**Comment by JianjunJin** [comment]
The quick answer for the current GetOrganelle is no.

There may be degenerated consensus or fragments depending on the data. You may also like to check this out: https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#why-does-getorganelle-generate-a-circular-genome-or-not-for-embplant_nrfungus_nr

**Comment by DKoenemann** [comment]
Thanks for the response!
